prov/verbs/src/fi_verbs.h

```diff
@@ -956,7 +969,34 @@ vrb_send_iov_flags(struct vrb_ep *ep, struct ibv_send_wr *wr,
 	if (flags & FI_FENCE)
 		wr->send_flags |= IBV_SEND_FENCE;
 
-	return vrb_post_send(ep, wr, flags);
+	if (wr->send_flags & IBV_SEND_INLINE && hmem != FI_HMEM_SYSTEM) {
+		bounce_buf = malloc(len);
+		if (!bounce_buf)
+			return -FI_ENOMEM;
```

Comment:
Even in the hmem case, we don't want the overhead of using malloc on every transfer.  Use alloca if the inline size is small enough (should be), or pull from a buffer pool.

Reply:
Sounds good.